distribution/docs/src/main/resources/content/_configuring/configuring-rest-for-users.adoc

```diff
@@ -9,5 +9,28 @@
 
 If using REST services or connecting to REST sources, several configuration options are available.
 
-${branding} includes a <<{managing-prefix}configuring_included_idp,Identity Provider(IdP)>>, but can also be configured to support an <<{managing-prefix}connecting_to_an_external_idp,external IdP>> or <<{managing-prefix}configuring_without_an_idp,no IdP>> at all.
+${branding} includes an <<{managing-prefix}configuring_included_idp,Identity Provider (IdP)>>, but can also be configured to support an <<{managing-prefix}connecting_to_an_external_idp,external IdP>> or <<{managing-prefix}configuring_without_an_idp,no IdP>> at all.
+The following diagram shows the configurations options.
 
+.REST Services Configuartion Options
+[ditaa, rest_config_options, png, ${image-width}]
+....
++------------------------------------------------------------------------------------------------------------+
+|                                       /---------------------------\                                        |
+|                                       | Configuring REST Services |                                        |
+|                   /-------------------|         for Users         |-------------------\                    |
+|                   |                   \---------------------------/                   |                    |
+|                   |                                 |                                 |                    |
+|                   v                                 v                                 v                    |
+|       /----------------------\        /---------------------------\         /-----------------------\      |
+|       | Configuring Included |        | Connecting to an External |         | Configuring without an|      |
+|    /--|   Identity Provider  |---\    |      Identity Provider    |     /---|   Identity Provider   |--\   |
+|    |  \----------------------/   |    \---------------------------/     |   \-----------------------/  |   |
+|    v                             v                                      v                              v   |
+| /--------------\    /---------------\                             /-------------\      /-----------------\ |
+| | Configuring  |    | Connecting to |                             |  Using STS  |      |  Connecting to  | |
+| | Included STS |    | External STS  |                             | without IdP |      | an External STS | |
+| \--------------/    \---------------/                             \-------------/      |   without IdP   | |
+|                                                                                        \-----------------/ |
++------------------------------------------------------------------------------------------------------------+
```

Comment:
💯

Reply:
✏️ It seems weird to me to switch between `IdP` and `Identity Provider` in the diagram

Reply:
That's because they are the section titles